### PR TITLE
TOOLS/PROFILE: Fix incorrect assert

### PR DESCRIPTION
**Pull request by dmitrygx**
## What

Move the incorrect assert to the proper place in ucx_read_profile tool code

## Why ?

Fixes the following assertion failure:
```
ucx_read_profile: read_profile.c:558: show_profile_data_log: Assertion `reqid_ctr > 1' failed.
```

## How ?

1. Move `assert()` under the case when khash entry was found
2. Remove extra empty space after `NEW` in the `"NEW "` string constant
3. Fix name of `num_records` variable
4. Use `size_t` instead of `int` for request ID to avoid overflow

### CODE REVIEW

File: src/tools/profile/read_profile.c [hunk 1]
```diff
@@ -553,9 +553,8 @@ static void show_profile_data_log(profile_data_t *data, options_t *opts,
                     reqid = reqid_ctr++;
                     kh_value(&reqids, hash_it) = reqid;
                 }
-                action = "NEW ";
+                action = "NEW";
             } else {
-                assert(reqid_ctr > 1);
```
Comment (brminich) [inline src/tools/profile/read_profile.c]: it was added for
`UCS/DATASTRUCT, TOOLS/PROFILE: Suppress clang false-positive warnings`

Comment (dmitrygx) [reply src/tools/profile/read_profile.c]: yes, static analyzer detects this problem again in Azure - will re-investigate

Comment (dmitrygx) [reply src/tools/profile/read_profile.c]: @brminich it has to be moved under the handler when `kh_get()` is able to find request - `kh_value()` has to be successful in this case

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>UNKNOWN</b> on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 25-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/56e9385bc756dec2f7d0bb1a467436943c11111c/_main) | :question: ABORTED
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/916fad9a15319f9001bf1209b07e029da84bbce3/hpc-arm-hwi-jenkins_W3) | :question: ABORTED
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/1ecfbacd07424dea8940890bd84bfbe3228a9eb7/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/075f8bf9f81c371d4f3db34168cadaa2c1c715f9/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/b7ba9f2147978dd4a0953c5bfe0eaeee423f5320/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/00701c94936a1d4dedc3e33fe319da96e5e73557/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/93bfb1f1b71d6ef9ce0a5340d4960b2046e3ac3f/hpc-arm-cavium-jenkins_W0) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/7cb6373ce3e77a694ae7e85db58d631a7b85848d/hpc-arm-cavium-jenkins_W1) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/710a8f42b1cd16e4da7898ae45ee2f57e222d581/hpc-arm-cavium-jenkins_W2) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/01b901849e5b4acbc18532d344d83287af6ab8a2/hpc-arm-cavium-jenkins_W3) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/95e2fefd005d48d273f46490d81edaeddb0aa121/hpc-arm-hwi-jenkins_W0) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/54f9d8c1d7aea1a6a07cb3c786e79461afd77fe9/hpc-arm-hwi-jenkins_W1) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/7ade5736ca545dac7184f0026f3da1518312f7e1/hpc-arm-hwi-jenkins_W2) | :question: UNKNOWN
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/d541f6fe7c1ee2c24d6be6825d3255320944fb1a/hpc-test-node-gpu_W0) | :question: UNKNOWN
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/614b676f98827dc038934cbe1edfdfab39222ed0/hpc-test-node-gpu_W1) | :question: UNKNOWN
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/e7f41e2d551aff656aa5b91500cb1e0d5e70357c/hpc-test-node-gpu_W2) | :question: UNKNOWN
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/8b67dc64d742ff5a6c95c0331ff00c46ecf56853/hpc-test-node-gpu_W3) | :question: UNKNOWN
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/94ab3211e3e96312f98b806f132235fdbdbe02c2/hpc-test-node-legacy_W0) | :question: UNKNOWN
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/88c4ef3f0259af99c1b591c85fee3228e99b5bc5/hpc-test-node-legacy_W1) | :question: UNKNOWN
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/63047abcebd12143a03fbe40eaf92c696d6961cb/hpc-test-node-legacy_W2) | :question: UNKNOWN
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/7feb2ea07701e0ac867ffe316a49ce7415070cdb/hpc-test-node-legacy_W3) | :question: UNKNOWN
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/f6d6302748373a2a499cdb4af9f57c1c324c8ea9/hpc-test-node-new_W0) | :question: UNKNOWN
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/87aadea8e548f25c4519b866ed55ac13bc270f8e/hpc-test-node-new_W1) | :question: UNKNOWN
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/2865815932bfd75d097698305a61b27d59f7da9e/hpc-test-node-new_W2) | :question: UNKNOWN
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/a64b18e514dfa608b3576f4dc3f2b758/raw/873c7ebe106850a89a4e82ca88d8675b4b3290f8/hpc-test-node-new_W3) | :question: UNKNOWN

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 25-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/42006d99c1bec894deef24e6daf688e88917af79/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/5ef4d6a5d2a3380693985ed2b6feb9f2cbe2a6fb/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/c0d5fdd7fdd9f120c7b7d8cbb5e4673b38f31221/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/c0d731db45040133be1e8f630f0c66bea71c4c73/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/7cf0b95fd53ceb5f6ae02c2d479220ab52621141/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/6f6e9121af6368ca0dd891aaf8bf9fc7ed141123/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/a8a6e75b3e41d53b54b51a5145e4ca05473e1450/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/c4ecf4053f262291c06d936c2eb989d8af672531/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/83fa0565926ae2765adb2598882eae2e38916699/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/b7c315217915faca2d6d76dc954175b522e651d2/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/133329e62535362ef1d352561b7f93eece010487/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/c10e0440102b60eda55fcdffae2d3350df898673/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/f2373fec7959e9e0f30d63fed008458db8351753/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/c123d1a10ef07a429de84f6b8503fefde174410e/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/a761f5548190ebda94b0c89d2cb2ad4aea763db5/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/61992d42f0d78ecc2d3bc8f911d7c1e3441e06c2/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/bf955f1e8503d7270a427f9389ec8de49794a991/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/65bd49ee4b34a0df4099f19cfa6e29169240de6f/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/17cfae8a6ffc39f1b939b022b99e3777a6bf3421/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/c5f9a23f2118ea819badbf9da4167d70c25bfb58/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/97a1d7be60b7c93666c0438f1e235433c78b23dd/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/1ff809a6309a85957785f5edddd8e8334606cd4c/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/2e7951892db874f3756dcc5171f499b13812f159/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/a4bd9f94286d75792644d72aacf0df72e98252dd/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/19fe73adbde295aa04e2b0300a68f7aa/raw/fd3f343f86aaa1b301f1e0a88f276dca171d4fb4/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by dmitrygx** [comment]
@brminich @hoopoepg could you review pls?

File: src/tools/profile/read_profile.c [hunk 2]
```diff
@@ -423,13 +423,14 @@ static int show_profile_data_accum(profile_data_t *data, options_t *opts)
     return ret;
 }
 
-KHASH_MAP_INIT_INT64(request_ids, int)
+KHASH_MAP_INIT_INT64(request_ids, size_t)
```
Comment (brminich) [inline src/tools/profile/read_profile.c]: uint64_t?
looks like this type is stored in the hash

File: src/tools/profile/read_profile.c [hunk 3]
```diff
@@ -441,7 +442,7 @@ static void show_profile_data_log(profile_data_t *data, options_t *opts,
     khash_t(request_ids) reqids;
     int hash_extra_status;
     khiter_t hash_it;
-    int reqid, reqid_ctr = 1;
+    size_t reqid;
```
Comment (brminich) [inline src/tools/profile/read_profile.c]: uint64_t?

File: src/tools/profile/read_profile.c (hunk 2 above)
Comment (dmitrygx) [reply src/tools/profile/read_profile.c]: `INT64` in the name of the auxiliary macro is a type of the key
so, we can use any type with it. I just aligned it with the type of `num_records`

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 25-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/3142f23512b5bb502f6fe5808ba49da15e6c96a3/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/baddde3dbf0d3e8747623e6e6c13b93a2894d6e0/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/d1713eac5918809224be4ff323504f668efa663d/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/3e4fb379975d0f1f1d4b8adc8ec786358df52afa/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/03503922bf757d49b2793f5a8f353fbc38715a9e/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/dd1c51dbdb1baa814810e50a8e3911794957bc76/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/af59a6a8abba1ba8a42501271409f82152a1a116/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/4bbebd99722e982c6a79be6cf945f16ddaacab6a/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/91320528ab3ef66cce3ef3051599aad4185632b5/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/8a563088f510136338069e4345cde0d0af3852cb/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/9d28cf51fc65877300727edae6520546e0a8eaee/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/3d1b35d40c62fc2d572d70a395d8fd5fc4705732/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/2309a69c60d82ea04b60da043952fd19b913398a/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/28c7a1cff47ebe71c549c3543c231013231fd7e2/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/a7320f5103bbac47388ef543b792cf89b4968988/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/77d26a2ca4a013d3c8bce0d085ea34f6ad468d2b/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/066516762b3c00b7f32ade8f073be461b8498c80/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/b9e744a5bc8a795c193470923c5c14438148309d/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/88777efe21b436259ef14d13e5175d1529f2d515/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/764e620b7518c3bbcac528ac6c669026354cef69/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/dcf82b968b036920c7dd471f003ba6d9a798b35d/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/a8ab2a547055d2cc5f1990d4bc12b6bbc2de9b39/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/80dd7b82f98c024ae744d7d25abc319dc9a1761c/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/c7cdf03fcbd2f168b8ace38d661fb5121da39410/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f88150fb5a09656625960f71b10d2b0a/raw/61b22307cc9a75078f5f29f5628d0fe0dc3e20ea/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: src/tools/profile/read_profile.c (hunk 2 above)
Comment (brminich) [reply src/tools/profile/read_profile.c]: the point is that the value which is stored in the hash is uint64_t

**Comment by dmitrygx** [comment]
@yosefe could you review pls?

**Pull request merged**